app/views/webchat/SelfEmploymentIncomeSupportView.scala.html

```diff
@@ -1,48 +0,0 @@
-@*
- * Copyright 2022 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *@
-
-@import config.AppConfig
-@import uk.gov.hmrc.webchat.client.WebChatClient
-@import views.html.templates.MainTemplate
-
-@this(main_template: MainTemplate,
-    appConfig: AppConfig,
-    webChatClient: WebChatClient)
-
-@()(implicit request: Request[_], messages: play.api.i18n.Messages)
-
-@main_template(title = messages("self.employment.income.support.scheme.title")) {
-
-    <h1 class="header-margin-top">@messages("self.employment.income.support.scheme.title")</h1>
-
-    <div class="govuk-inset-text">
-        <p>@messages("global.opening.times.title")</p>
-        <p>@messages("global.opening.times.8to7_30")</p>
-        <p>@messages("global.opening.times.weekend.p")</p>
-    </div>
-
-    <h2 class="margin-top">@messages("global.webchat.title.h2")</h2>
-
-    @components.webchat_container(appConfig, webChatClient)
-    @components.webchat_loading_text()
-
-    <p class="margin-top">
-        <a class="govuk-link" id="return-link" href=@appConfig.selfEmploymentIncomeSupportReturnUrl>
```

Comment:
I don't think the selfEmploymentIncomeSupportReturnUrl in app.config is used anywhere else. Can you please check and remove from appconfig if required